### PR TITLE
[DOCS-9424] Error tracking-specific setup docs

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

- Adds a tab for Error Tracking in Browser Setup docs

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/real_user_monitoring/browser/setup/client.md [hunk 1]
```diff
@@ -2020,17 +2178,210 @@ See `usePartitionedCrossSiteSessionCookie`.
...
+: Optional - **Deprecated**<br/>
+**Type**: Boolean<br/>
+**Default**:`false`<br/>
+See `usePartitionedCrossSiteSessionCookie`.
+
+`allowFallbackToLocalStorage`
+: Optional<br/>
+**Type**: Boolean<br/>
+**Default**: `false`<br/>
+Allows the use of `localStorage` when cookies cannot be set. This enables the Browser SDK to run in environments that do not provide cookie support. See [Monitor Electron Applications Using the Browser SDK][13] for a typical use-case.
```
Comment (jhgilbert) [inline content/en/real_user_monitoring/browser/setup/client.md:2354]:
```suggestion
Allows the use of `localStorage` when cookies cannot be set. This enables the Browser SDK to run in environments that do not provide cookie support. See [Monitor Electron Applications Using the Browser SDK][13] for a typical use case.
```

**Review by jhgilbert — APPROVED** [review]
Approved with a minor suggestion, thank you!

File: content/en/real_user_monitoring/browser/setup/_index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: RUM Browser Monitoring Setup
+title: Browser Monitoring Setup
```
Comment (nulrich) [inline content/en/real_user_monitoring/browser/setup/_index.md:2]: Just noticed that in some place like [here](https://github.com/DataDog/documentation/blob/a7e0f18152a23b60da2b9e8fe5869a20d5d74088/content/en/real_user_monitoring/browser/setup/server.md) it's still `RUM Browser Monitoring`.

Comment (aliciascott) [reply content/en/real_user_monitoring/browser/setup/_index.md:2]: ok thank you I'll get that fixed

**Pull request merged**